docassemble/MotionToStayEviction/data/questions/review_screen.yml

```
@@ -0,0 +1,386 @@
+
+      Other fee paid to the court: ${ word(yesno(fees["Other"].waive)) }
+
+      Amount (if known): ${ showifdef('fees["Other"].amount') }
+
+      Describe the other fee/cost: ${ showifdef('fees["Other"].description') }
+
+      Other special fees or costs: ${ word(yesno(fees["Other cost"].waive)) }
+
+      Amount (if known): ${ showifdef('fees["Other cost"].amount') }
+
+      Describe the other fee/cost: ${ showifdef('fees["Other cost"].description') }
+  - Edit: |-
+      dont_know_docket_number
+    button: |
+      **Docket Number**
+
+      I do not know the docket number: ${ word(yesno(dont_know_docket_number)) }
+
+      Docket number: ${ showifdef('docket_numbers[0]') }
+
+      What case do you want fees waived for? Briefly explain how the clerk can find your case.: ${ showifdef('docket_numbers[0]') }
+  - Edit: |-
+      housing_court
+    button: |
+      **Select the Housing Court department and division where your case was heard.**
+
+      ${ showifdef('housing_court') }
+  - Edit: |-
+      lower_court_case.docket_number
+    button: |
+      **What is the Housing Court docket number for your case?**
+
+      Docket number: ${ showifdef('lower_court_case.docket_number') }
+  - Edit: |-
+      lower_court_case.judgment_appealing
+    button: |
+      **What judgment are you appealing?**
+
+      ${ showifdef('lower_court_case.judgment_appealing') }
+
+      Other: ${ showifdef('other_judgment_appealing') }
+  - Edit: |-
+      lower_court_case.judgment_date
+    button: |
+      **When did the Housing Court enter judgment against you?**
+
+      Date: ${ showifdef('lower_court_case.judgment_date') }
+  - Edit: |-
+      noa_service_date
+    button: |
+      **When will you deliver copies of your Notice of Appeal to the plaintiffs' attorney(s)?**
+
+      Date: ${ showifdef('noa_service_date') }
+  - Edit: |-
+      user_wants_affidavit
+    button: |
+      **Do you want to submit an affidavit of indigency?**
+
+      Do you want to file an affidavit of indigency?: ${ word(yesno(user_wants_affidavit)) }
+  - Edit: |-
+      housing_court
+    button: |
+      **Select the Housing Court department and division where your case was heard.**
+
+      ${ showifdef('housing_court') }
+  - Edit: |-
+      lower_court_case.docket_number
+    button: |
+      **What is the Housing Court docket number for your case?**
+
+      Docket number: ${ showifdef('lower_court_case.docket_number') }
+  - Edit: |-
+      is_initial_filing
+    button: |
+      **Is this Motion to Stay the first document you are filing in the Appeals Court?**
+
+      ${ word(yesno(is_initial_filing)) }
+  - Edit: |-
+      appeals_court_docket_number
+    button: |
+      **What is the **Appeals Court** docket number for your case?**
+
+      ${ showifdef('appeals_court_docket_number') }
+  - Edit: |-
+      forty_eight_hour_notice
+    button: |
+      **Have you received a {48-hour notice}?**
```

Unfortunately, that term doesn't show up here, should probably remove the brackets.
```suggestion
      **Have you received a 48-hour notice?**
```